### ISSUE TITLE
[question] conan info doesnt take into accountpython requires dependency

### ISSUE THREAD

**Issue by sandeshhardikar** [conan-io/conan#9277]
I have a recipe which has build_requires , requires and python_requires .

conan info doesnt track "python_requires" neither in dependency graph or in the text output .

Is there any other way to get list of "python_requires" dependency ?

Thank in advance ,
Sandesh

**Comment by memsharded** [comment]
I think the lockfile contains the locked python_requires.

I guess it might be doable adding them to ``conan info`` command, but unless there is a very compelling and demanded use case, that would probably need to wait until Conan 2.0 new CLI.

**Comment by sandeshhardikar** [comment]
Thank you for your response
yes , the locked file contains python_requires .

But passing the lock file to `conan info` has no effect on the output . It still gives the same output and python_requires is missing in it .

```sh
conan lock create conanfile.py --lockfile-out=dummy.lock
conan info path_to_recipe -l dummy.lock --only None
```
My use Case : 
We want to provide some system level packages to our customer , which in turn are dependent on lot of other packages . We want to determine programmatically which packages have to be uploaded outside the organization when we make a release . `conan info` was the suitable command for this , except for missing python_requires package

**Comment by memsharded** [comment]
Understood.

I have proposed a PR in https://github.com/conan-io/conan/pull/9290 to add this information to ``conan info`` output (both stdout and json, that I guess you are interested in if doing automatic processing).

**Comment by memsharded** [comment]
Merged, this will be in next 1.39
